koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -504,9 +505,8 @@ def convert_csv_to_parquet(self, csv_filename):  # noqa: C901
             return None, None, False
 
         daily_data_frames = []
-        _, csv_name = os.path.split(csv_filename)
-        parquet_file = None
-        parquet_base_filename = csv_name.replace(self.file_extension, "")
+        parquet_filepath = ""
+        parquet_base_filename = csv_filename.name.replace(self.file_extension, "")
```

Comment:
```suggestion
        parquet_base_filename = csv_filename.stem
```

Reply:
I did try this initially, but the `.csv.gz` throws this off, otherwise, yes, I would accept this suggestion!

Reply:
Yeah, I wondered about that. Compound extensions do throw a wrench in things.